src/NuGet.Status/Web.config

```diff
@@ -11,19 +11,40 @@
     <add key="UnobtrusiveJavaScriptEnabled" value="true" />
 
     <!--Settings-->
-    <add key="Config:FileName" value="dev-usnc" />
-  </appSettings>
-  <!--
-    For a description of web.config changes see http://go.microsoft.com/fwlink/?LinkId=235367.
+    <add key="Storage:ConnectionString" value="UseDevelopmentStorage=true" />
+    <add key="Storage:SecondaryConnectionString" value="UseDevelopmentStorage=true" />
+    <add key="Storage:ContainerName" value="status" />
+    <add key="Storage:BlobName" value="status.json" />
+    <add key="Storage:TableName" value="status" />
 
-    The following attributes can be set on the <httpRuntime> tag.
-      <system.Web>
-        <httpRuntime targetFramework="4.6.2" />
-      </system.Web>
-  -->
+    <add key="KeyVault:VaultName" value="" />
+    <add key="KeyVault:CertificateThumbprint" value="" />
+    <add key="KeyVault:ClientId" value="" />
+    <add key="KeyVault:StoreLocation" value="" />
+    <add key="KeyVault:StoreName" value="" />
+    <add key="KeyVault:ValidateCertificate" value="false" />
+    
+    <add key="NuGetBaseUrl" value="https://dev.nugettest.org" />
+    
+    <add key="Ida:AdminEnabled" value="false" />
+    <add key="Ida:RootUri" value="" />
+    <add key="Ida:AADInstance" value="" />
+    <add key="Ida:ClientId" value="" />
+    <add key="Ida:Tenant" value="" />
+    
+    <add key="ExternalBrandingMessage" value="&#169; Microsoft 2017" />
```

Comment:
change this to 2019?

Reply:
Yup. The actual change is in `NuGetDeployment` though.